frontend/src/metabase/actions/containers/ActionPicker/ActionPicker.tsx

```diff
@@ -142,17 +154,15 @@ function ModelActionPicker({
   );
 }
 
-const ConnectedModelActionPicker = Actions.loadList({
-  query: (state: State, props: { model: { id: number | null } }) => ({
-    "model-id": props?.model?.id,
+export const ConnectedActionPicker = _.compose(
+  Search.loadList({
+    query: () => ({
+      models: ["dataset"],
+    }),
+    loadingAndErrorWrapper: true,
+    listName: "models",
   }),
```

Comment:
nit: I think we can remove `loadingAndErrorWrapper: true` here because it's enabled by default anyway